cryptoinvestor/views/investments.py

```diff
@@ -0,0 +1,27 @@
+from cryptoinvestor.views import BaseView
+
+
+class InvestmentsListView(BaseView):
+    def get_template_name(self):
+        return 'investments/investments.html'
+
+    def get_objects(self):
+
+        context = {}
+        investments = self.app.user.account.investments
+
+        base = self.app.local_currency
+
+        for key in investments:
+            asset = self.app.assets.get(key)
+            rate = asset.rates.get(base, []).get('rate', 'N/A')
+
+            for investment in investments[key]:
+                addition = 100 if investment['amount'] < 0 else -100
+                investment['profit'] = round(((rate / investment['price']) * 100) + addition, 2)
```

Comment:
I don't quite understand how you can have 200% profit while selling at the same value as buying.

Buy profit seems good. But on sell profit it would be good to have something like flat value gained, meaning:

```python
profit_on_sell = (rate_now * amount) - (rate_at_buy * amount)
```

With plus and minus signs and appropriate colors of course